index.html

```diff
@@ -100,172 +66,122 @@ <h3>Supporting beneficial decentralization technologies.</h3>
 		</div>
 	</section>
 
-	<!-- video -->
-	<div class="video-container">
-		<div class="video-wrapper">
-			<!-- <iframe width="700" height="400" src="//www.youtube.com/embed/7QLaKW8ABy4?&amp;hd=1&amp;rel=0&amp;showinfo=0;theme=light;color=whitee" frameborder="0" allowfullscreen></iframe> -->
-			<!-- Original dimensions are 1280x720, so 704x396 is exact same ratio -->
-			<video width="704" height="396" id="demoVideo"
-				class="video-js vjs-default-skin vjs-big-play-centered"
-				controls preload="none"
-				poster="other/okTDemoPoster.jpg"
-				data-setup='{}'
-			>
-				<source src="other/okTurtlesDEMO.webm">
-				<source src="other/okTurtlesDEMO.mp4">
-				<source src="other/okTurtlesDEMO.ogv">
-				<source src="other/okTurtlesDEMO.mov">
-				<p class="vjs-no-js">
-			    To view this video please enable JavaScript, and consider upgrading to a web browser that supports HTML5 video.
-			  </p>
-			</video>
-		</div>
-	</div>
-
```

Comment:
@pieer instead of deleting all of the JS & HTML related to the video, can you comment it out so that it's easily available in case we to add another video?